server/server.js

```diff
@@ -371,6 +373,28 @@ app.post(`${apiBaseUrl}/employer-sites`,
     }
   }));
 
+app.patch(`${apiBaseUrl}/employer-sites/:id`,
+  keycloak.allowRolesMiddleware('ministry_of_health'),
+  keycloak.getUserInfoMiddleware(),
+  asyncMiddleware(async (req, res) => {
+    await validate(EditSiteSchema, req.body);
```

Comment:
A little meta, but it seems `history` is included in the validation. I don't think that should be allowed to be edited.